agenta-backend/agenta_backend/services/evaluators_service.py

```diff
@@ -24,23 +29,24 @@ def auto_similarity_match(variant_output, correct_answer, settings_values) -> Re
     return result
 
 
-def auto_regex_test(test_string, regex, should_match):
-    re_pattern = re.compile(regex, re.IGNORECASE)
-    result = bool(re_pattern.search(test_string))
-    return result == should_match
+def auto_regex_test(_, test_string: str, settings_values: dict) -> Result:
+    re_pattern = re.compile(settings_values["regex_pattern"], re.IGNORECASE)
+    result = (
+        bool(re_pattern.search(test_string)) == settings_values["regex_should_match"]
+    )
+    return Result(type="bool", value=result)
 
 
 def evaluate(
-    evaluator_name,
-    correct_answer,
-    variant_output,
-    settings_values,
-    *additional_args,
-    **additional_kwargs,
+    evaluator_name: str,
+    correct_answer: str,
+    variant_output: str,
+    settings_values: dict,
```

Comment:
I think when typing we should use Dict (from typing import Dict) and not dict

Reply:
@aybruhm